yarn-project/simulator/src/avm/opcodes/contract.ts

```diff
@@ -39,12 +39,13 @@ export class GetContractInstance extends Instruction {
       instance.deployer.toField(),
       instance.contractClassId,
       instance.initializationHash,
-      instance.publicKeysHash,
+      // This this okay ?
+      ...instance.publicKeys.toFields(),
```

Comment:
This could be a problem down the line, depending on what the type of the infinity flag (U1, U8, FF) is expected to be. 

We shouldnt need to solve it now though